### ISSUE TITLE
[Improving] - define target

### ISSUE THREAD

**Issue by AndrewArch** [w3c/wai-planning-and-implementation#67]
It seldom makes sense to make a "A" and then a "AA" pass at pages to address the same issue. Maybe we can rephrase this to take about addressing critical issues (at all levels) and then addressing remaining issues later?

**Comment by nitedog** [comment]
Agreed. Changed to:

> For example, meet particular WCAG 2.0 success criteria in the coming release, and meet Level AA success criteria in the following release.
